3-enrich/stream-enrich/examples/config.hocon.sample

```diff
@@ -41,6 +43,27 @@ enrich {
     secret-key: "iam"
   }
 
+  # config for NSQ sink and NSQ source
+NSQ {
+  #Topic name for good event source
+  good-source-topic: "{{sinkKinesisNsqSourceGoodTopicName}}"
+
+  #Channel name for good event source
+  good-source-channel: "{{sinkKinesisNsqSourceGoodChannelName}}"
+
+  #Topic name for bad event sink
+  bad-sink: "{{sinkKinesisNsqSinkTopicBadName}}"
```

Comment:
There are channel fields in the NSQ additionally. I thought that keeping channel and topic name separated can be confusing therefore I put all of the topic and channel fields under NSQ section.